libs/server/Resp/BasicCommands.cs

```diff
@@ -753,16 +753,6 @@ private bool NetworkIncrement<TGarnetApi>(byte* ptr, RespCommand cmd, ref TGarne
                 *(valPtr + RespInputHeader.Size) = (byte)'1';
                 input = new ArgSlice(valPtr, vsize);
             }
-            else if (cmd == RespCommand.DECR)
```

Comment:
We better leave this as it was so in the case where the key does not exist we do a direct upsert of "-1".

Reply:
emm....
https://github.com/microsoft/garnet/blob/03611e19292bd27c0a2c185a33279b3c429f7010/libs/server/Storage/Functions/MainStore/VarLenInputMethods.cs#L130-L143
Because of this part, I modified the command.